SetupGuide_Plugin_Admin.php

```diff
@@ -700,8 +700,8 @@ public function get_browsercache_settings() {
 	 * @uses $_POST['enable']
 	 */
 	public function config_browsercache() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
-			$enable               = ! empty( $_POST['enable'] ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the two phpcs comments to have it validate the changes.

Reply:
Removed 2 PHPCS disable comments.